### ISSUE TITLE
Karma fails on startup

### ISSUE THREAD

**Issue by tbrd** [karma-runner/karma#1325]
Occasionally, typically after Karma has exited unexpectedly or I've stopped it with a CTRL+C, Karma refuses to start up with the following error: 
[TypeError: "Cannot call method 'kill' of null"]

Requires a machine restart :(

This has been an issue for me for some time, but happens very rarely. A colleague with the same setup has been seeing it more often.

Command:
karma --browsers PhantomJS --reporters dots start

Setup:
 karma@0.12.31
 karma-chrome-launcher@0.1.7
 karma-coverage@0.2.7
 karma-firefox-launcher@0.1.4
 karma-jasmine@0.2.3
 karma-phantomjs-launcher@0.1.4
 karma-requirejs@0.2.2
 karma-safari-launcher@0.1.1

Config:
// Karma configuration
// Generated on Mon Apr 08 2013 15:25:02 GMT-0700 (PDT)

module.exports = function(karma) {
    karma.set({
        // base path, that will be used to resolve files and exclude
        basePath: './',

```
    frameworks: ['requirejs', 'jasmine'],

    // list of files / patterns to load in the browser
    files: [
        'client/web/scripts/swift/test/mocks/mock_td.js',
        'client/web/scripts/swift/test/mocks/file_reader.js',
        'client/web/scripts/lib/bower_components/jquery/dist/jquery.js',
        'client/web/scripts/lib/bower_components/jasmine-jquery/lib/jasmine-jquery.js',
        'client/web/scripts/lib/bower_components/jasmine-flight/lib/jasmine-flight.js',
        {pattern: 'client/web/scripts/lib/bower_components/lodash/dist/lodash.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/twitter-text/twitter-text.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/es5-shim/es5-shim.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/sinonjs-built/pkg/sinon-1.8.2.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/jjv/lib/jjv.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/requirejs-text/text.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/flight-with-child-components/lib/flight-with-child-components.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/flight-with-resources/lib/with-resources.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/flight-with-state/lib/with-state.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/es6-promise/promise.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/rxjs/dist/rx.lite.min.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/Array.prototype.find/index.js', included: false},
        {pattern: 'client/web/scripts/swift/test/utils/observable_test_utils.js', included: false},
        {pattern: 'client/web/scripts/schema/*.json', included: false},
        'client/web/scripts/core/underscore.strings-min.js',
        'client/web/scripts/core/jq.pubsub.js',
        'client/web/scripts/lib/bower_components/hogan/lib/template.js',
        'client/web/scripts/lib/bower_components/hogan/lib/compiler.js',
        'client/web/scripts/lib/jquery.*.js',
        'client/web/scripts/lib/bootstrap-*.js',
        'client/web/scripts/lib/antiscroll/antiscroll.js',
        {pattern: 'client/web/scripts/core/defer.js', included: false},
        {pattern: 'client/web/scripts/controller/decider.js', included: false},
        {pattern: 'client/web/mustaches/{*/,}/*.mustache', included: false},
        {pattern: 'client/web/scripts/UI/template.js', included: false},
        {pattern: 'client/web/scripts/UI/columns.js', included: false},
        {pattern: 'client/web/scripts/i.js', included: false},
        {pattern: 'client/web/scripts/assets.js', included: false},
        {pattern: 'client/web/scripts/storage/store.js', included: false},
        {pattern: 'client/web/scripts/storage/memory_storage.js', included: false},
        {pattern: 'client/web/scripts/sync/util.js', included: false},
        {pattern: 'client/web/scripts/utils/i18n.js', included: false},
        {pattern: 'client/web/scripts/swift/app/ui/**/*.js', included: false},
        {pattern: 'client/web/scripts/swift/app/page/**/*.js', included: false},
        {pattern: 'client/web/scripts/swift/app/tracking/**/*.js', included: false},
        {pattern: 'client/web/scripts/swift/app/data/**/*.js', included: false},
        {pattern: 'client/web/scripts/swift/app/util/**/*.js', included: false},
        {pattern: 'client/web/scripts/lib/bower_components/flight/**/*.js', included: false},
        {pattern: 'client/web/scripts/swift/test/spec/**/*.spec.js', included: false},
        {pattern: 'client/web/scripts/swift/test/spec/fixtures/*.html', included: false},
        'client/web/scripts/swift/app/karma-require-config.js'
    ],

    // use dots reporter, as travis terminal does not support escaping sequences
    // possible values: 'dots', 'progress', 'junit', 'teamcity'
    // CLI --reporters progress
    reporters: ['dots', 'coverage'],

    // preprocessors to apply before executing the suite
    preprocessors: {
        // Only measure 'new' swift application code since we don't run
        // qunit with karma.
        'client/web/scripts/swift/app/**/*.js': ['coverage']
    },

    coverageReporter: {
        type: 'cobertura',
        dir: 'coverage/'
    },

    // list of files to exclude
    exclude: [],

    // web server port
    // CLI --port 9876
    port: 9876,

    // cli runner port
    // CLI --runner-port 9100
    runnerPort: 9100,

    // enable / disable colors in the output (reporters and logs)
    // CLI --colors --no-colors
    colors: true,

    // level of logging
    // possible values: karma.LOG_DISABLE || karma.LOG_ERROR || karma.LOG_WARN || karma.LOG_INFO || karma.LOG_DEBUG
    // CLI --log-level debug
    logLevel: karma.LOG_ERROR,

    // enable / disable watching file and executing tests whenever any file changes
    // CLI --auto-watch --no-auto-watch
    autoWatch: true,

    // Start these browsers, currently available:
    // - Chrome
    // - ChromeCanary
    // - Firefox
    // - Opera
    // - Safari (only Mac)
    // - PhantomJS
    // - IE (only Windows)
    // CLI --browsers Chrome,Firefox,Safari
    browsers: ['Chrome', 'Firefox', 'Safari', 'PhantomJS'],

    // Manually set viewport size
    // (Note: currently does not work, waiting on
    // https://github.com/ariya/phantomjs/issues/10619 to be resolved)
    customLaunchers: {
      'PhantomJS_custom': {
        base: 'PhantomJS',
        options: {
          viewportSize: { width: 480, height: 800 }
        }
      }
    },

    // If browser does not capture in given timeout [ms], kill it
    // CLI --capture-timeout 5000
    captureTimeout: 5000,

    // Auto run tests on start (when browsers are captured) and exit
    // CLI --single-run --no-single-run
    singleRun: false,

    // report which specs are slower than 500ms
    // CLI --report-slower-than 500
    reportSlowerThan: 500,

    plugins: [
        'karma-chrome-launcher',
        'karma-coverage',
        'karma-firefox-launcher',
        'karma-jasmine',
        'karma-phantomjs-launcher',
        'karma-requirejs',
        'karma-safari-launcher'
    ]
});
```

};

**Comment by devoto13** [comment]
This sounds like a valid bug to me. However, there have been many changes to the libraries since it was reported. Please open a new issue including a minimal reproduction if you still experience this issue.
